packages/terra-tabs/CHANGELOG.md

```diff
@@ -4,18 +4,22 @@ ChangeLog
 Unreleased
 ----------
 
-6.2.0 - (June 12, 2019)
-------------------
-### Removed
-* Removed node_modules from .npmignore
-
 ### Changed
 * Update tests for terra-toolkit v5 and terra-dev-site v5 changes
 
+6.2.0 - (June 12, 2019)
+------------------
 ### Added
 * box-shadow theme variable for in-active tab: --terra-tabs-structural-box-shadow
 * z-index theme variable for active tab: --terra-tabs-structural-active-z-index
 
+### Changed
+* Update tests for terra-toolkit v5 and terra-dev-site v5 changes
```

Comment:
is there a mis-merge here?

Reply:
fixed: https://github.com/cerner/terra-framework/pull/729/commits/eb13cb5cbec7a5c6633d9bcbb60a5407dac2e60c